tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -773,3 +794,14 @@ user gets resolved comment
 user closes Set Page View box
     user clicks element    id:pageViewToggleButton
     user waits until element is not visible    id:editingMode    %{WAIT_SMALL}
+
+# This keyword will work for any URL containing the pattern release/<guid>.    This keyword will return the matched
+# guid.    For example, in the URL https://localhost/publication/<publication id>/release/<release id>/status, this
```

Comment:
```suggestion
# guid. For example, in the URL https://localhost/publication/<publication id>/release/<release id>/status, this
```

Reply:
I thought this was the Robot formatter doing this, but upon changing it and committing again, it now seems to be behaving itself, so this is updated too thanks!